src/client/entity-editor/creator-data.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2016  Ben Ockmore
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+
+import AliasEditor from './alias-editor/alias-editor';
+import CreatorData from './creator-data/creator-data';
+import IdentifierEditor from './identifier-editor/identifier-editor';
+import {Panel} from 'react-bootstrap';
+import React from 'react';
+import SharedData from './shared-data/shared-data';
+import {connect} from 'react-redux';
+
+
+let Wrapper = ({
```

Comment:
Not sure I like the name `Wrapper`. If that's all this is, I wonder if it should be split into multiple components or subsumed by another.